src/main/java/Duke.java

```diff
@@ -1,10 +1,177 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    private static final String horizontalBorder = "_________________________________\n";
+    private static final String INVALID_TODO_INPUT = " ☹ OOPS!!! The description of a todo cannot be empty.\n";
+    private static final String INVALID_DEADLINE_INPUT = "☹ OOPS!!! Please use proper deadline formatting: deadline {task} /by {time}\n";
+    private static final String INVALID_EVENT_INPUT = "☹ OOPS!!! Please use proper event formatting: event {task} /at {time}\n";
+    private static final String INVALID_ACCESS_EMPTY_TASKLIST = "☹ OOPS!!! Task does not exist. Initialise a task first, then try again\n";
+    private static final String INVALID_USER_INPUT = "☹ OOPS!!! Please use one of these keywords: {deadline, event, todo} followed by \\\"by\\\" and \\\"at\\\" for deadline and event tasks respectively.\n";
+
+    private Tasklist tasklist;
+
+    private Duke(){
+        this.tasklist = new Tasklist();
+    }
+
+    private Duke(Tasklist tasklist){
+        this.tasklist = tasklist;
+    }
+
+    private static String welcomeMessage(){
+        return horizontalBorder + "Hello! I'm Duke\nWhat can I do for you?\n" + horizontalBorder;
+    }
+
+    private static String byeMessage(){
+        return horizontalBorder + "Bye. Hope to see you again soon!\n" + horizontalBorder;
+    }
+
+    private String listContents(){
+        return horizontalBorder + this.tasklist + horizontalBorder;
+    }
+
+    public String addTaskMessage(String taskString) {
+        return horizontalBorder +  "Got it. I've added this task:\n" + taskString + "\n" + "Now you have " + this.tasklist.getCount() + " tasks in the list.\n" + horizontalBorder;
```

Comment:
Line length for this line is pretty long so you could look into using line wrapping such as splitting at the addition operators.